buổi-10/main.js

```diff
@@ -0,0 +1,150 @@
+// bài 1
+const getMultiplication = number => {
+    if (number >= 1 && number <= 10) {
+        for(let i = 1; i <= 10; i++) {
+            console.log(number + ' * ' + i + ' = ' + (number * i))
+        } 
+    } else {
+        console.log('Chỉ nhận giá trị từ 1-> 10!!')
+    }
```

Comment:
indent là 2 dấu cách (1 tab = 2 spaces)
Mỗi 1 tab sai, thừa/thiếu 1 space được tính là 1 comment
```suggestion
  } else {
    console.log('Chỉ nhận giá trị từ 1-> 10!!')
  }
```